recipe/bld.bat

```diff
@@ -1,5 +1,5 @@
 :: There is no `gdal-config` on Windows so we need to hardcode gdal's version.
-set GDAL_VERSION=2.4.2
```

Comment:
Nice catch!

Reply:
It sucks that we need that and it is easy to forget to update it. Luckily it only affects Windows :smile: